lambda/otel/Dockerfile

```diff
@@ -13,24 +13,32 @@ RUN bundle config set --local path 'ruby'
 # Save gem filename if present
 RUN ls | grep solarwinds_apm-*.gem > gem_file.txt 2>/dev/null || true
 
-# Install dependencies
-RUN bundle update
+# (for local gem only)
+RUN gem_file=$(cat gem_file.txt) && \
+    if [ -f "$gem_file" ]; then \
+        gem install $gem_file; \
```

Comment:
i don't get why we install here, then unpack below. seems just unpack is needed if all we want is to bundle the source files into the layer.